vue-frontend/src/components/partials/ScreenHeader.vue

```diff
@@ -63,15 +87,28 @@ export default {
     return {
       headerLogoImage: headerLogoImage,
       careerURL: "/jobs",
+      contactURL: "/contact",
       blogsURL: Config.BLOG_URL,
       navbarSticky: false,
       navbarAnimation: false,
       navContainerHeight: 0,
       lastScrollY: 0,
+      activeCareerPath: false,
+      activeContactPath: false,
     };
   },
   components: {},
   mounted() {
+    let currentRoutePath = this.$router.currentRoute._value.path;
+
+    currentRoutePath == this.careerURL
```

Comment:
`this.activeCareerPath = currentRoutePath == this.careerURL`
We can write it like above shorthand